src/ui/public/agg_types/__tests__/metrics/parent_pipeline.js

```diff
@@ -0,0 +1,144 @@
+import _ from 'lodash';
+import expect from 'expect.js';
+import ngMock from 'ng_mock';
+import DerivativeProvider from 'ui/agg_types/metrics/derivative';
+import CumulativeSumProvider from 'ui/agg_types/metrics/cumulative_sum';
+import MovingAvgProvider from 'ui/agg_types/metrics/moving_avg';
+import SerialDiffProvider from 'ui/agg_types/metrics/serial_diff';
+import VisProvider from 'ui/vis';
+import StubbedIndexPattern from 'fixtures/stubbed_logstash_index_pattern';
+
+const metrics = [
```

Comment:
in the UI, consider using the same division that you used for pipeline aggs. it presents well.